src/main/java/com/thecookiezen/bussiness/cluster/entity/DockerHost.java

```diff
@@ -6,19 +6,23 @@
 
 import java.io.Serializable;
 import java.time.Instant;
+import java.util.Random;
 
 @Data
-@NoArgsConstructor
 public class DockerHost implements Serializable {
     private static final long serialVersionUID = 1L;
 
-    private long id;
+    private final long id;
```

Comment:
\[Checkstyle\] WARNING: 'member def modifier' have incorrect indentation level 4, expected level should be 2\.